sfxclient/wrapper.go

```diff
@@ -18,7 +18,7 @@ func NewMultiCollector(collectors ...Collector) Collector {
 
 // Datapoints returns the datapoints from every collector.
 func (m MultiCollector) Datapoints() []*datapoint.Datapoint {
-	ret := make([]*datapoint.Datapoint, len(m))
+	ret := make([]*datapoint.Datapoint, 0)
```

Comment:
Would `ret := make([]*datapoint.Datapoint, 0, len(m))` be better?

Reply:
`len(m)` is totally irrelevant to the actual number of datapoints, unless you were sure that each collector would only have at most one datapoint.